Library/Homebrew/formula.rb

```diff
@@ -1008,7 +1008,7 @@ def caveats
   # rarely, you don't want your library symlinked into the main prefix
   # see gettext.rb for an example
   def keg_only?
-    keg_only_reason && keg_only_reason.valid?
+    !!keg_only_reason && keg_only_reason.valid?
```

Comment:
I'm confused now, wouldn't `keg_only` without arguments actually fail? The method signature is `keg_only(reason, explanation = "")`.